data_store/messaging/tf_messaging.py

```diff
@@ -343,13 +355,21 @@ def _invalid_enum_value_failure_message(self, validation_failure: InvalidEnumVal
             project_number = get_project_number_by_position(validation_failure.row_index, validation_failure.table)
             section = f"Project Funding Profiles - Project {project_number}"
 
-        cell_index = self._construct_cell_index(
-            table=validation_failure.table,
-            column=validation_failure.column,
-            row_index=validation_failure.row_index,
+        # if column is a str make it a list
+        columns = (
+            [validation_failure.column] if isinstance(validation_failure.column, str) else validation_failure.column
```

Comment:
Should we make `validation_failure.column` actually just be `validation_failure.columns` and only `list[str]`? Did you look into that and it's a pain?

Reply:
I considered it a few times changing class definitions. But it's quite an intimidating change to make.
We should look into all these classes, but perhaps as a separate exercise when we focus on this specifically.

Personally I am rather tempted to have `column` always as a `str`. Out of all validation failure types only one is breaking the pattern.